src/browser/types.ts

```diff
@@ -26,6 +26,8 @@ declare global {
 
             extendOptions(opts: { [name: string]: unknown }): Promise<void>;
 
+            getConfig(): Promise<BrowserConfig>;
```

Comment:
It looks like as `fix`, because it should have been described when write this file